lms/resources/_js_config/__init__.py

```diff
@@ -572,6 +572,10 @@ def _config(self):
             config["debug"]["tags"].append(
                 "role:instructor" if self._lti_user.is_instructor else "role:learner"
             )
+            config["user"] = {
```

Comment:
I called the group "user", but that might not be clear enough. Other options are `current_user`, `active_user`, `session_user`, etc.

I'm open to suggestions.

Reply:
I think `user` is fine. It seems obvious enough that it refers to the logged-in user.